lib/initial-setup/WP_Auth0_InitialSetup.php

```diff
@@ -107,14 +66,13 @@ public function render_setup_page() {
 
 	public function cant_create_client_message() {
 		?>
-		  <div id="message" class="error">
+		  <div class="notice notice-error">
```

Comment:
Where are these CSS classes defined? I didn't see them anywhere in the CSS of this lib.

Reply:
Core WP admin classes. I pulled them out of an existing, core error message. Good eyes, though!